src/nn/context.ts

```diff
@@ -110,4 +110,11 @@ export class MLContext {
         tf.version_core}/dist/`);
     return tf;
   }
+
+  /** @internal */
+  // Expose wasm interface for supporting configure threads for wasm backend.
+  //     wasm.setThreadsCount(n)
+  get wasm(): unknown {
+    return wasm;
```

Comment:
Where does the `wasm` object comes from? Could you please explain how `setThreadsCount()` works？

Reply:
> Where does the wasm object comes from?

On [Line-4 of src/nn/context.ts](https://github.com/webmachinelearning/webnn-polyfill/blob/master/src/nn/context.ts#L4), 
```js
import * as wasm from '@tensorflow/tfjs-backend-wasm';
```
here, `wasm` represents a namespace object which contains all exports from  **@tensorflow/tfjs-backend-wasm** module.

> Could you please explain how `setThreadsCount()` works？

`setThreadsCount()` must be called before calling `tf.setBackend('wasm')`, please see [this setting part](https://github.com/webmachinelearning/webnn-polyfill/pull/208/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5) in README file.

Reply:
Thanks for the explanation.